2025-03-11 13:37:31,525	INFO worker.py:1841 -- Started a local Ray instance.
[36m(PoolActor pid=20444)[0m *** SIGSEGV received at time=1741714688 on cpu 20 ***
[36m(PoolActor pid=20444)[0m PC: @     0x145c943e4cfe  (unknown)  ray::gcs::TaskInfoAccessor::AsyncAddTaskEventData()
[36m(PoolActor pid=20444)[0m     @     0x145ca3063d10       1792  (unknown)
[36m(PoolActor pid=20444)[0m     @     0x145c942c7a45       1392  ray::core::worker::TaskEventBufferImpl::FlushEvents()
[36m(PoolActor pid=20444)[0m     @     0x145c94253a0c       1488  ray::core::CoreWorker::Disconnect()
[36m(PoolActor pid=20444)[0m     @     0x145c94253dbd       1152  ray::core::CoreWorker::ForceExit()
[36m(PoolActor pid=20444)[0m     @     0x145c942541ef       1680  ray::core::CoreWorker::HandleKillActor()
[36m(PoolActor pid=20444)[0m     @     0x145c9424b6f4        192  ray::rpc::ServerCallImpl<>::HandleRequestImpl()
[36m(PoolActor pid=20444)[0m     @     0x145c9459a128       1168  EventTracker::RecordExecution()
[36m(PoolActor pid=20444)[0m     @     0x145c945950fe         48  std::_Function_handler<>::_M_invoke()
[36m(PoolActor pid=20444)[0m     @     0x145c94595576        112  boost::asio::detail::completion_handler<>::do_complete()
[36m(PoolActor pid=20444)[0m     @     0x145c94b6acbb        128  boost::asio::detail::scheduler::do_run_one()
[36m(PoolActor pid=20444)[0m     @     0x145c94b6c639        288  boost::asio::detail::scheduler::run()
[36m(PoolActor pid=20444)[0m     @     0x145c94b6cd42         96  boost::asio::io_context::run()
[36m(PoolActor pid=20444)[0m     @     0x145c94190b81       1280  ray::core::CoreWorker::RunIOService()
[36m(PoolActor pid=20444)[0m     @     0x145c9464b890         64  thread_proxy
[36m(PoolActor pid=20444)[0m     @     0x145ca30591ca  (unknown)  start_thread
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460: *** SIGSEGV received at time=1741714688 on cpu 20 ***
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460: PC: @     0x145c943e4cfe  (unknown)  ray::gcs::TaskInfoAccessor::AsyncAddTaskEventData()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145ca3063d10       1792  (unknown)
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c942c7a45       1392  ray::core::worker::TaskEventBufferImpl::FlushEvents()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c94253a0c       1488  ray::core::CoreWorker::Disconnect()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c94253dbd       1152  ray::core::CoreWorker::ForceExit()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c942541ef       1680  ray::core::CoreWorker::HandleKillActor()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c9424b6f4        192  ray::rpc::ServerCallImpl<>::HandleRequestImpl()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c9459a128       1168  EventTracker::RecordExecution()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c945950fe         48  std::_Function_handler<>::_M_invoke()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c94595576        112  boost::asio::detail::completion_handler<>::do_complete()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c94b6acbb        128  boost::asio::detail::scheduler::do_run_one()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c94b6c639        288  boost::asio::detail::scheduler::run()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c94b6cd42         96  boost::asio::io_context::run()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c94190b81       1280  ray::core::CoreWorker::RunIOService()
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145c9464b890         64  thread_proxy
[36m(PoolActor pid=20444)[0m [2025-03-11 13:38:08,914 E 20444 20477] logging.cc:460:     @     0x145ca30591ca  (unknown)  start_thread
[36m(PoolActor pid=20444)[0m Fatal Python error: Segmentation fault
[36m(PoolActor pid=20444)[0m 
[36m(PoolActor pid=20444)[0m 
[36m(PoolActor pid=20444)[0m Extension modules: msgpack._cmsgpack, google._upb._message, psutil._psutil_linux, psutil._psutil_posix, setproctitle, yaml._yaml, _cffi_backend, ray._raylet, pvectorc, mkl._mklinit, mkl._py_mkl_service, numpy.core._multiarray_umath, numpy.core._multiarray_tests, numpy.linalg.lapack_lite, numpy.linalg._umath_linalg, numpy.fft._pocketfft_internal, numpy.random._common, numpy.random.bit_generator, numpy.random._bounded_integers, numpy.random._mt19937, numpy.random.mtrand, numpy.random._philox, numpy.random._pcg64, numpy.random._sfc64, numpy.random._generator (total: 25)
Map: random_32_32_10 requires 2 runs
Map: warehouse_10_20_10_2_2 requires 4 runs
[36m(checkIfRunNextAgents pid=20257)[0m Completed map: random_32_32_10, agentNum: 50, combined_df: (2, 44)
[36m(checkIfRunNextAgents pid=20257)[0m 2 2 50 random_32_32_10
[36m(checkIfRunNextAgents pid=20257)[0m Completed map: random_32_32_10, agentNum: 100, combined_df: (4, 44)
[36m(checkIfRunNextAgents pid=20257)[0m 2 2 100 random_32_32_10
[36m(checkIfRunNextAgents pid=20257)[0m Finished all agent numbers for map: random_32_32_10
[36m(checkIfRunNextAgents pid=20197)[0m 4 4 50 warehouse_10_20_10_2_2
------------ Finished eecbs Calls in :12.902 seconds
[36m(checkIfRunNextAgents pid=20197)[0m 4 4 100 warehouse_10_20_10_2_2
------------ Finished Data Manipulator in :32.282 seconds
[36m(checkIfRunNextAgents pid=20197)[0m Completed map: warehouse_10_20_10_2_2, agentNum: 100, combined_df: (8, 44)[32m [repeated 2x across cluster] (Ray deduplicates logs by default. Set RAY_DEDUP_LOGS=0 to disable log deduplication, or see https://docs.ray.io/en/master/ray-observability/user-guides/configure-logging.html#log-deduplication for more options.)[0m
[36m(checkIfRunNextAgents pid=20197)[0m Finished all agent numbers for map: warehouse_10_20_10_2_2
2025-03-11 13:38:14,599	INFO worker.py:1841 -- Started a local Ray instance.
Map: random_32_32_10 requires 2 runs
Map: warehouse_10_20_10_2_2 requires 4 runs
[36m(checkIfRunNextAgents pid=21091)[0m Finished all scens for map: random_32_32_10
------------ Finished eecbs Calls in :29.614 seconds
[36m(checkIfRunNextAgents pid=21091)[0m Finished all scens for map: warehouse_10_20_10_2_2
------------ Finished Data Manipulator in :9.577 seconds
